conf/base/bblayers.conf

```diff
@@ -18,7 +18,7 @@ BBLAYERS += "${BSPDIR}/sources/meta-uav"
 BBLAYERS += "${BSPDIR}/sources/meta-ros"
 BBLAYERS += "${BSPDIR}/sources/meta-intel-aero"
 BBLAYERS += "${BSPDIR}/sources/meta-intel-aero-connectivity"
-BBLAYERS += "${BSPDIR}/sources/meta-intel-realsense"
+BBLAYERS += "${BSPDIR}/sources/meta-intel-aero-librealsense"
```

Comment:
A `repo sync` should fix that since the changes to the manifest were pushed. I just rebuilt everything from scratch and it's working fine.